### ISSUE TITLE
Tag docker images with version

### ISSUE THREAD

**Issue by jmattheis** [SinusBot/docker#30]
It would be great, if the docker images could get tagged with a version, something like `sinusbot/docker:1.0.0`. This has several benefits. 
* Consistent version of the bot between different servers
* Maybe a bug gets introduced in a newer version, then an older version can be choosen
* Docker best practice
   > When building images, always tag them with useful tags which codify version information, intended destination (prod or test, for instance), stability, or other information that is useful when deploying the application in different environments. Do not rely on the automatically-created latest tag.
   
   https://docs.docker.com/develop/dev-best-practices/

**Comment by irgendwr** [comment]
Hi,

yes this is indeed best practice but since we where in alpha/beta phase before we generally **only** wanted people to run the latest release since older releases do generally contain more bugs and miss many new features that the new scripts require to work.

Since the latest release of 1.0.0-beta.1 however we're moving into stable territory and tagging the versions is beginning to make sense.

Of course we will still **only support the latest release** at the moment and **the SinusBot should always be kept up to date** since regressions are rare, especially in the stable-isch non-insider releases and scripting updates are important.

**TL;DR:** yes, soon™️ 

*Buuut...* old versions expire after a certain amount of time so you will not be able to run them after they reach a certain age. This will not change.

/cc @flyth

**Comment by irgendwr** [comment]
Added version tags, you can view the complete here: https://hub.docker.com/r/sinusbot/docker/tags

Note to quay.io users:
*These tags are **not** available on quay.io yet. If someone actually cares about that for some reason: remind me to fix it.*
Why do people even care about quay.io?

Note to maintainers:
*This now requires manually building and pushing with the `./release.sh <version>` helper script (use `docker login` before running it).
It would make sense to automate this via CI - I don't really care right now though sooo have fun.*
